src/config/mod.rs

```diff
@@ -2988,6 +2988,10 @@ impl TikvConfig {
                 .to_owned();
         }
 
+        if self.storage.engine == EngineType::RaftKv2 {
```

Comment:
Why?

Reply:
raftstore 2 always use async write, and the configuration is by default 0.